src/components/responsive-flyout/CloseButton.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Button from '../button';
+import IconClose from '../../icons/general/IconClose';
+import { bdlGray62 } from '../../styles/variables';
+import './styles/CloseButton.scss';
+
+export type CloseButtonProps = {
+    className?: string,
+    onClick?: Function,
+};
+
+type Props = CloseButtonProps;
+
+const CloseButton = ({ className, onClick }: Props) => {
+    return (
+        <Button onClick={onClick} className={classNames('bdl-close-btn', className)} type="button">
```

Comment:
Ideally, this would be named in a Flyout-specific way to ensure it's scoped properly. We also want to use [SUIT naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), so maybe `bdl-ResponsiveFlyoutClose`